### PR TITLE
Travis CI: Add .travis.yml

### PR DESCRIPTION
**Pull request by gaocegege-caicloud**

### CODE REVIEW

**Comment by gaocegege-caicloud** [comment]
https://travis-ci.org/gaocegege-caicloud/kubeflow-clientset/builds/317358097

PTAL, now Travis CI checks if the code generated by code-generator is updated and returns ERRCODE 1 if not.

File: hack/update-codegen.sh [hunk 1]
```diff
@@ -19,7 +19,7 @@ set -o nounset
 set -o pipefail
 
 ROOT=$(dirname ${BASH_SOURCE})/..
-BOILERPLATE=$(realpath ${ROOT})/hack/custom-boilerplate.go.txt
+BOILERPLATE=${ROOT}/hack/custom-boilerplate.go.txt
```
Comment (gaocegege) [inline hack/update-codegen.sh]: The code is changed because it is unnecessary and returns `realpath: command not found` in Travis CI

Comment (DjangoPeng) [reply hack/update-codegen.sh]: Have you dryrun or test the PR in Travis CI?

Comment (gaocegege) [reply hack/update-codegen.sh]: https://travis-ci.org/gaocegege-caicloud/kubeflow-clientset/builds/317358097

@gaocegege-caicloud @DjangoPeng

Comment (gaocegege) [reply hack/update-codegen.sh]: And, we need a owner of caicloud to enable the repo in Travis CI.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+language: go
+
+go:
+  - 1.9
```
Comment (Jimexist) [inline .travis.yml]: also add 1.8 please

**Comment by Jimexist** [comment]
add to readme

```
[![Build Status](https://travis-ci.org/caicloud/kubeflow-clientset.svg?branch=master)](https://travis-ci.org/caicloud/kubeflow-clientset)
```

**Comment by gaocegege** [comment]
PTAL @Jimexist @DjangoPeng

**Comment by DjangoPeng** [comment]
LGTM, merged.

**Pull request merged**